pocs/utils/images/fits.py

```diff
@@ -0,0 +1,322 @@
+import os
+import shutil
+import subprocess
+
+from warnings import warn
+
+from astropy.io import fits
+from astropy.wcs import WCS
+from astropy import units as u
+
+from pocs.utils import error
+
+
+def solve_field(fname, timeout=15, solve_opts=[], **kwargs):
+    """ Plate solves an image.
+
+    Args:
+        fname(str, required):       Filename to solve in .fits extension.
```

Comment:
Astrometry's `solve_field` supports a variety of input image formats, including JPEG and PNG, not just FITS. Perhaps `solve_field()` and `get_solve_field()` should be in a location other than the fits specific file?

Reply:
The `solve_field` program does support more options but our wrappers around them do not. In the exiting files there was some old code for how we handle `CR2`, but none of that was relevant any more so I removed it for this PR.

The `solve_field()` we have could be more generic, but `get_solve_field()` definitely assumes FITS (despite some of the comments in the docstring, which I just noticed and will clean).

We could put `solve_field` somewhere else and then change `get_solve_field` to `solve_fits_field`.